### ISSUE TITLE
Fix Line Overflow in LaTeX templates

### ISSUE THREAD

**Issue by IvanIsCoding** [IvanIsCoding/ResuLLMe#2]
Sometimes, if a line is too long it can overflow. Some cases that trigger it:
* Very long given name
* Very long university name
* Lots of skills in a category

The templates that are most affected by it are:
* Deedy
* Plush
* Simple

**Comment by harsiddhdave44** [comment]
@IvanIsCoding I'd like to take this one.
Can you please add some screenshots of this issue?

**Comment by IvanIsCoding** [comment]
@karankhatavkar yes, the issue is open. I don’t have screenshots but it is not very hard to trigger.
If you use the example JSON (https://github.com/IvanIsCoding/ResuLLMe/blob/main/.github/Turing.json) and replace the university name for something longer e.g. “University of Illinois at Urbana-Champaign” it should trigger it
